src/modules/Packages.rb

```diff
@@ -1550,6 +1571,7 @@ def AdjustSourcePropertiesAccordingToProduct(src_id)
         sources_set = []
         Builtins.foreach(sources_got) do |one_source|
           if Ops.get_integer(one_source, "SrcId", -1) == src_id
+            log.info("Updating repository name from \"#{repo_name}\" to \"#{new_name}\"")
```

Comment:
another option is to use inspect and not need double quote. It also helps if name is nil.
```ruby
irb(main):001:0> nil.to_s
=> ""
irb(main):002:0> nil.inspect
=> "nil"
```

Reply:
Initially I used `#inspect` but then Rubocop complained about a too long line. :smiley:
So I changed that to a shorter form. And `nil` should not happen here, pkg-bindings would already complain earlier...